src/App.vue

```diff
@@ -20,29 +44,68 @@
                 />
               </v-col>
             </v-row>
-          </v-col>
-         
+          </v-col>         
         </v-row>
+
       </v-container>
     </v-main>
   </v-app>
+
 </template>
 
+
 <script>
 
   import IndicatorPanel from './components/IndicatorPanel.vue'
+
   import stats from './data/sample.json'
 
   export default {
 
     components:{
-      IndicatorPanel
+      IndicatorPanel,
     },
 
     data () {
       return {
-        clusters: stats.clusters
+        apiclusters: null,
+        clusters: stats.clusters,
+        canary: { 
```

Comment:
This canary info/details should be coming from the API

Reply:
Noted